docs/operator-manual/ingress.md

```diff
@@ -188,6 +188,7 @@ metadata:
   annotations:
     nginx.ingress.kubernetes.io/force-ssl-redirect: "true"
     nginx.ingress.kubernetes.io/ssl-passthrough: "true"
+    nginx.ingress.kubernetes.io/backend-protocol: "HTTPS"
```

Comment:
```suggestion
```

Since SSL passthrough forwards the traffic without decrypting it, the NGINX Ingress Controller does not need to know whether the backend uses HTTPS. The annotation `nginx.ingress.kubernetes.io/backend-protocol: "HTTPS"` is irrelevant in this scenario because the traffic remains encrypted as it passes through the ingress.